keras_preprocessing/image/dataframe_iterator.py

```diff
@@ -44,7 +44,7 @@ class DataFrameIterator(BatchFromFilesMixin, Iterator):
         target_size: tuple of integers, dimensions to resize input images to.
         color_mode: One of `"rgb"`, `"rgba"`, `"grayscale"`.
             Color mode to read images.
-        classes: Optional list of strings, classes to use (e.g. `["dogs", "cats"]`).
+        classes: Optional numpy array/list of strings, classes to use (e.g. `["dogs", "cats"]`).
```

Comment:
All Iterable would work right?

Reply:
Yes! Should I change it to "Optional `Iterable` with strings, ..."?